libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -158,7 +186,7 @@ internal class RouteRefreshController(
             currentLegIndex,
         )
         if (routeDiffs.isEmpty()) {
-            logI("No changes to route annotations", LOG_CATEGORY)
+            logI("No changes in annotations for route ${newRoute.id}", LOG_CATEGORY)
```

Comment:
Square brackets look like brackets from `listOf().toString()` . Why do you suggest them?

Reply:
without bracket, it's not obvious where's a template and where injected data

Reply:
Sorry, I still don't understand why it's so important to have brackets. Do you mean that for some people it could be hard to understand following line in the logs
```
No changes in annotations for route d6c1ed04-9d84-4cfe-b841-7f625c6a2df4#1
```
but they will understand the next one without problems?
```
No changes in annotations for route [d6c1ed04-9d84-4cfe-b841-7f625c6a2df4#1]
```

What do you think about concern that square brackets look like lists(`emptyList().toString()`) and could create confusion?

Reply:
Discussed internally, will introduce it in a separate PR.